ai-service/src/main/resources/application.yml

```
@@ -0,0 +1,49 @@
+    password: root1234
+    driver-class-name: org.postgresql.Driver
+
+  # JPA / Hibernate 설정
+  jpa:
+    database-platform: org.hibernate.dialect.PostgreSQLDialect
+    hibernate:
+      ddl-auto: update
+    show-sql: true
+    properties:
+      hibernate:
+        format_sql: true # SQL을 보기 좋게 출력하려면 추가하세요
```

YAMLlint (1.38.0)

[warning] too few spaces before comment: expected 2

(comments)